web/wp-content/themes/dctx-v2/functions.php

```diff
@@ -0,0 +1,241 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-nlid9r.php, but found phpcs-scan-nLid9r.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).